fv3core/README.md

```diff
@@ -194,6 +196,54 @@ Add `PULL=False` to build from scratch without running `docker pull`:
 PULL=False make build
 ```
 
+### Bare-Metal
+
+Pace requires gcc > 9.2, cuda, MPI, and Python 3.8 on your system in order to run bare-metal.
```

Comment:
Am I missing something? This mentions `pace` but it's in the `fv3core`-specific `README.md`.

Reply:
Good question, we've only really run the dycore bare-metal on gaea, so it felt a little uncomfortable to put the build instructions under the Pace readme, but we could put it elsewhere

Reply:
I think something like this belongs under top-level documentation. At some point we should start sphinx docs, the readme is getting very large.